SOC/common/simulation.py

```diff
@@ -95,11 +95,20 @@ def plot_state(self, with_boundaries = False):
         Plots the current state of the simulation.
         """
         fig, ax = plt.subplots()
+
         if with_boundaries:
             values = self.values
         else:
             values = self.values[self.BOUNDARY_SIZE:-self.BOUNDARY_SIZE, self.BOUNDARY_SIZE:-self.BOUNDARY_SIZE]
+        
+        values_on_plot=False
```

Comment:
To zdecydowanie powinno być opcjonalnym argumentem jak `with_boundaries = False)` w linijce 93 :)